### PR TITLE
Adds clear action example to amp-selector demos.

### PR DESCRIPTION
**Pull request by mianuddin**
This pull request accompanies [this one](https://github.com/ampproject/amphtml/pull/10545) on the `amphtml` repository, which adds the `clear` action to `amp-selector`.

-Adds `clear` action example to `amp-selector` demos.

Reviewer: @aghassemi

### CODE REVIEW

File: src/20_Components/amp-selector.html [hunk 1]
```diff
@@ -86,6 +86,22 @@
...
+  <!-- ## Actions -->
+  <!--
+  The `clear` action may be used to clear all selections of an `amp-selector`.
+  -->
+  <div>
+    <button on="tap:clearActionDemoSelector.clear"
+            class="ampstart-btn ampstart-btn-secondary caps">
+            Clear Selection</button>
+    <amp-selector id='clearActionDemoSelector' layout="container" multiple>
+      <amp-img src="/img/landscape_sea_300x199.jpg"  width="60" height="40" option="1"></amp-img>
```
Comment (sebastianbenz) [inline src/20_Components/amp-selector.html]: Can you pre-select one of the items?

Comment (mianuddin) [inline src/20_Components/amp-selector.html]: @sebastianbenz No problem!

**Comment by aghassemi** [comment]
The feature on the AMP core side is merged, it didn't make it to Canary this week so ETA for prod is August 1. (we probably should wait until then before merging this PR)

**Comment by aghassemi** [comment]
@sebastianbenz This is in production now. We can merge.

**Comment by sebastianbenz** [comment]
Thanks everyone!

**Pull request merged**